go.mod

```diff
@@ -4,8 +4,8 @@ go 1.21
 
 require (
 	github.com/giantswarm/apiextensions-application v0.6.1
-	github.com/giantswarm/cluster-standup-teardown v1.0.3
-	github.com/giantswarm/clustertest v0.20.0
+	github.com/giantswarm/cluster-standup-teardown v1.1.1-0.20240513000831-b7c9d5dd7da3
+	github.com/giantswarm/clustertest v0.20.1-0.20240512235412-f3c924c97b9b
```

Comment:
Update after https://github.com/giantswarm/clustertest/pull/181 is merged and released.